pkg/admission/pathannotation/pathannotation_admission.go

```diff
@@ -101,20 +105,20 @@ func (p *pathAnnotationPlugin) Admit(ctx context.Context, a admission.Attributes
 		return nil
 	}
 
-	this, err := p.logicalClusterLister.Cluster(clusterName).Get(corev1alpha1.LogicalClusterName)
+	cluster, err := p.getLogicalCluster(clusterName, corev1alpha1.LogicalClusterName)
```

Comment:
super-nit: can you make these `logicalCluster` please? xref #2524

Reply:
it looks like you already did it in another PR :)